app/lib/fisbot/failure_group_api.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class FailureGroupApi < FisbotApi
+  class << self
+    def resource
+      :failure_groups
+    end
+
+    def failure_group_description(job_id)
+      uri = api_access.resource_uri(job_id.to_s)
+      array = JSON.parse(Net::HTTP.get(uri))
+      return array['description'] if array.present?
+    end
+  end
+end
```

Comment:
Similar to how we have designed the `CodeLocation` model, rename this to `FailureGroup`. Rename the function at line 9 to `description`.